codc/codc.py

```diff
@@ -46,7 +45,7 @@ class PatchedLib:
     def __init__(self):
 
         ffi.cdef(self.__read_header())
-        self.__lib = ffi.dlopen('libodccore.so')
+        self.__lib = ffi.dlopen('libodccore')
```

Comment:
@simondsmart, after reading up a bit on dlopen() I've learned that this utility can figure out the proper file suffix (.dylib, .so, etc) as well as do proper searching (LD_LIBRARY_PATH, etc.). I think the fix then becomes very simple which is to delete the __find_library() routine that I started, and simply call dlopen() without the file suffix.

I tested this on my Mac and on Ubuntu and it works on those two systems.

Does this solution sound okay with you?

Thanks!